packages/builder/src/components/common/RoleSelect.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+  import { Select } from "@budibase/bbui"
+  import { roles } from "stores/backend"
+  import { RoleUtils } from "@budibase/frontend-core"
+
+  export let value
+  export let error
+  export let placeholder = null
+  export let autoWidth = false
+  export let quiet = false
+</script>
+
+<Select
+  {autoWidth}
+  {quiet}
+  bind:value
+  on:change
+  options={$roles}
+  getOptionLabel={role => role.name}
+  getOptionValue={role => role._id}
+  getOptionColour={role => RoleUtils.getRoleColour(role._id)}
+  {placeholder}
+  {error}
+/>
```

Comment:
Now that the new design UI is merged, we have this `RoleSelect` component already in `src/components/design/settings/controls/RoleSelect.svelte`. I coped that component into here since we expected this PR to be merged before the new design UI was. I'll jump on this branch and sort out this usage, and the role utils which I also copied from that new branch.

Reply:
Cool, thanks @aptkingston